docusaurus/.styles/Google/AMPM.yml

```diff
@@ -0,0 +1,9 @@
+extends: existence
+message: "Use 'AM' or 'PM' (preceded by a space)."
+link: 'https://developers.google.com/style/word-list'
+level: error
+nonword: true
+tokens:
+  - '\d{1,2}[AP]M'
+  - '\d{1,2} ?[ap]m'
+  - '\d{1,2} ?[aApP]\.[mM]\.'
```

Comment:
This is nice, will VSCode highlight warnings for all of these rules?

Reply:
https://github.com/errata-ai/vale-vscode